### PR TITLE
Restored max mappers option for Gobblin MR.

### PR DESCRIPTION
**Pull request by ibuenros**

### CODE REVIEW

**Comment by ibuenros** [comment]
@ydai1124 can you review?

File: gobblin-runtime/src/test/java/org/apache/hadoop/fs/FileSystemTestUtils.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+import java.io.IOException;
+import java.net.URI;
+
+import org.apache.hadoop.conf.Configuration;
+
+
+public class FileSystemTestUtils {
+
+  public static void addFileSystemForTest(URI uri, Configuration conf, FileSystem fs) throws IOException {
```
Comment (ydai1124) [inline gobblin-runtime/src/test/java/org/apache/hadoop/fs/FileSystemTestUtils.java]: Why is this necessary as a single line method?

Comment (ibuenros) [inline gobblin-runtime/src/test/java/org/apache/hadoop/fs/FileSystemTestUtils.java]: Because the method is protected, so it needs to be in the correct package (`org.apache.hadoop.fs`)

Comment (ydai1124) [inline gobblin-runtime/src/test/java/org/apache/hadoop/fs/FileSystemTestUtils.java]: I see.
